test/executor.test.js

```diff
@@ -175,6 +175,21 @@ describe('executor', function() {
     expect(app.models.Customer._modelsWhenAttached).to.include('UniqueName');
   });
 
+  it('throws on bad require() call inside boot script', function() {
+    appdir.writeFileSync('models/BadCustomer.js',
```

Comment:
Please follow the convention of keeping boot scripts in the `boot/` directory (not in `models/`).